optimade/models/jsonapi.py

```diff
@@ -225,22 +224,35 @@ class Resource(BaseModel):
     )
 
 
-class Success(BaseModel):
-    """A Successful response"""
+class Response(BaseModel):
+    """A top-level response"""
 
-    data: Union[None, Resource, Set[Resource]] = Schema(
+    data: Optional[Union[None, Resource, Set[Resource]]] = Schema(
         ..., description="Outputted Data"
     )
-    included: Optional[Set[Resource]] = Schema(
-        ..., description="A list of resources that are included"
-    )
-    meta: Optional[dict] = Schema(
+    meta: Optional[Meta] = Schema(
         ...,
         description="A meta object containing non-standard information related to the Success",
     )
-    links: Optional[Union[Links, Pagination]] = Schema(
-        ..., description="Information about the JSON API used"
+    errors: Optional[Set[Error]] = Schema(..., description="A list of errors")
+    included: Optional[Set[Resource]] = Schema(
+        ..., description="A list of resources that are included"
     )
-    jsonapi: Optional[JsonAPI] = Schema(
+    links: Optional[ToplevelLinks] = Schema(
```

Comment:
Although, referring to my previous comment on `ToplevelLinks`, this line will have to import it from `toplevel`, which I am not a fan of, and would definitely always want `jsonapi` to be a self-contained file, representing the JSON API v1.0.